pkg/api/client.go

```diff
@@ -325,7 +332,7 @@ func uploadResultsToCDS(c *Client, filesToUpload []string) error {
 			// log.Errorf("{warning:Unable to upload results %s\n}", err)
 			return err
 		}
-		log.Infof("Successfully uploaded to CDS: {info:%s}", guid)
+		log.Debugf("Successfully uploaded to CDS: {info:%s}", guid)
```

Comment:
I understand why you are changing this, but there is one issue so far. There is no way of tracking if we don't no the upload guid unfortunately till we get EVAL_GUID that is constant across to track.

Reply:
@hemanthgk10 I have kept this as info but made it a little more user friendly
```
		log.Infof("Assessment upload reference: {info:%s}", guid)
```